Combined release notes for Core Modules and Libs (#558)
SAP/cloud-sdk: docs/js/release-notes.mdx

- Update client libraries giving access to the latest SAP services on SAP Business Technology Platform and SAP S/4HANA.
- Protect yourself from bugs and breaking changes in the future

:::tip What are pregenerated typed client libraries (VDM)?

For your convenience we [pregenerate client libraries](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*) (also known as Virtual Data Model or VDM for short) for allow-listed OData services of SAP S/4HANA Cloud, Marketing Cloud, and SAP S/4HANA On-premise.
These libraries are not Open Source, but free and distributed under the SAP Developer license through [npmjs.com](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*).
You can always generate a typed client library yourself by using our [OData](features/odata/generate-odata-client.mdx) and [Open API](features/openapi/generate-openapi-client.mdx) code generators.

> To search for supported services use our [API Business Hub integration](https://blogs.sap.com/2021/09/17/the-sap-cloud-sdk-integrates-with-the-sap-api-business-hub/).

Reviewer: the term support services is not clear to me.

Author: ```suggestion
To easily search for services and get typed client library for them,  use our [API Business Hub integration](https://blogs.sap.com/2021/09/17/the-sap-cloud-sdk-integrates-with-the-sap-api-business-hub/).
```

Author: Good point, I tried to re-phrase it. We can give it another look later to make the heading of the page more structured.

:::

<!-- Copy and fill
## 1.xx.x [Core Modules] - January 1, 2021

> [Maven Central](https://search.maven.org/search?q=g:com.sap.cloud.sdk*%20AND%20v:3.34.1) | [Javadoc](https://help.sap.com/doc/55bb1642ea614bf582d514ed34401136/1.0/en-US/index.html)

Reviewer 2: Maven? 🧐

Author: ```suggestion
[GitHub](https://github.com/SAP/cloud-sdk-js/releases/tag/v1.52.0) | [npm](https://www.npmjs.com/search?q=%40sap-cloud-sdk)
```

Author: Good catch. The template is from Java. Fixed.

  - The update covers all the changes to existing OData services and introduces new ones.
- **We released a regular update for pregenerated client libraries for [Workflow API for Cloud Foundry](https://api.sap.com/api/SAP_CP_Workflow_CF/resource).**

## 1.52.0 [Core Modules] - November 5, 2021

[GitHub](https://github.com/SAP/cloud-sdk-js/releases/tag/v1.52.0) | [npm](https://www.npmjs.com/search?q=%40sap-cloud-sdk)

### Compatibility Notes

> - We changed default cache isolation strategy from `IsolationStrategy.Tenant` to `IsolationStrategy.Tenant_User`. This applies when you set `useCache` to `true` while using a `getDestination` method for a destination lookup.

Reviewer 3: ```suggestion
- We changed default cache isolation strategy from `IsolationStrategy.Tenant` to `IsolationStrategy.Tenant_User`. 
  This applies when you set `useCache` to `true` while using a `getDestination` method for a destination lookup.
```

### Fixed Issues

> - We now cache a destination only when a `JWT` contains all necessary information about a `Tenant` and a `User`. For example, when using `IsolationStrategy.Tenant_User`, the JWT has to contain both **tenant id** and **user id**.

Reviewer 3: ```suggestion
- We now cache a destination only when a `JWT` contains all necessary information about a `Tenant` and a `User`. 
  For example, when using `IsolationStrategy.Tenant_User`, the JWT has to contain both **tenant id** and **user id**.
```

> - If `JWT` has an expiration time we'll use it to override the default cache expiration time of 5 minutes.

Reviewer 3: ```suggestion
- If `JWT` has an expiration time, we'll use it to override the default cache expiration time of 5 minutes.
```
